FiveCalls/FiveCalls/Info.plist

```diff
@@ -19,22 +19,22 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.4</string>
+	<string>1.5</string>
```

Comment:
Don't forget that you also have to bump the extension version when you do this.

Reply:
Yup the submission warned me so I should have pushed that change afterwards.